### ISSUE TITLE
Unable to launch any Vulkan application

### ISSUE THREAD

**Issue by peterigz** [baldurk/renderdoc#3172]
## Description

I'm trying to use RenderDoc to debug my Vulkan renderer but it fails to launch (window closes immediately) and stops specifically at device creation with device lost validation error. In my efforts to get to the bottom of it I discovered that it's actually any Vulkan application. I tested with the vkcube app that comes with the Vulkan SDK and exactly the same thing, either with the pre-built binary in the bin folder or just compiling it myself with the source they provide in the demos folder. Fails at the same place - device lost at vkCreateDevice.

## Steps to reproduce

Launch the vkcube demo that comes with VulkanSDK. I strongly suspect that it's just my setup so really what I'm looking for is just some pointers to get to the bottom of the issue to figure out what might be conflicting if anything.

## Environment

* RenderDoc version: 1.30
* Operating System: Windows 11
* Graphics API: Vulkan
* Vulkan SDK version: 1.3.261.1

GPU: AMD Radeon 6600 with latest drivers.  All of the Radeon suite of GPU profiles connect and capture with no problems.

Thanks in advance!

**Comment by baldurk** [comment]
This may be an AMD driver bug, there are a few issues in the recent drivers on vulkan, though I haven't seen them happen in vkCreateDevice before. Can you try rolling back to a driver from 23.8.2 or earlier and testing that?

If that doesn't work, make sure that you do not have vkconfig running as that program can also cause crashes or bugs, and also try uninstalling or disabling any programs you have that may interfere with vulkan software - screen recorders, overlays, GPU performance monitors, etc

**Comment by peterigz** [comment]
I just tried with driver version 23.8.2 and same thing. But then it was vkconfig that I'd forgotten about that actually helped find the issue. I ran vkconfig and it threw up an error message about twitch overlay manifest not being valid, so I located and deleted that manifest and RenderDoc launches apps properly now. Thanks for the pointer, been stuck on this most of the day!

And also whilst testing I've noticed that the bug in my renderer that I was originally trying to fix no longer happens presumably because of the older driver version, so I guess that's a 2 in 1 👍
